src/data.ml

```diff
@@ -161,6 +161,12 @@ module People = struct
     uri       = Some "https://linse.me";
     email     = None;
   }
+
+  let mato = {
```

Comment:
Welcome to the Mirage blog at long last, @mato :-)

Reply:
@avsm Thanks! Can you do the honours and click the green button?

Reply:
Yep, was just checking through the infra first to make sure it's live for updates.